src/r_data.c

```diff
@@ -938,6 +918,8 @@ void R_InitTranMap(int progress)
   //
   int force_rebuild = M_CheckParm("-tranmap");
 
+  skytran = W_CacheLumpName("SKYTRAN", PU_CACHE);
```

Comment:
This *must* be PU_STATIC. Also, the variable could get moved to `r_plane.c`, made `static` and initialized in `R_InitPlanes()`.